.github/workflows/deploy-on-kind.yaml

```diff
@@ -0,0 +1,64 @@
+---
+name: Testing on Kind
+on:
+  push:
+    paths:
+      - 'charts/jenkins/**'
+  pull_request:
+    paths:
+      - 'charts/jenkins/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        kubernetes_version:
+          - "kindest/node:v1.20.0"
+          - "kindest/node:v1.19.1"
+          - "kindest/node:v1.18.8"
+          - "kindest/node:v1.17.11"
+          - "kindest/node:v1.16.15"
+      fail-fast: false
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.0.1
+
+      - name: Build Chart
+        run: helm dependency build charts/jenkins
+
+      - name: Run chart-testing (lint)
+        run: helm lint charts/jenkins
+
+      - name: Create Kind Cluster
+        uses: helm/kind-action@v1.1.0
+        with:
+          node_image: ${{ matrix.kubernetes_version }}
+
+      - name: Deploy Chart
+        run: helm upgrade --install --wait --set controller.ingress.enabled=true jenkins charts/jenkins
```

Comment:
If we want to test controller.ingress then it should be enabled in https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/ci/other-values.yaml

Reply:
I think that testing ingress would be a good idea, this is the area thats currently causing issues for people on 1.19+

Reply:
Sure! Every values file in ci folder is used for testing automatically.